arch/arm/boot/dts/overlays/README

```diff
@@ -3620,8 +3620,8 @@ Params: cma-512                 CMA is 512MB (needs 1GB)
         cma-default             Use upstream's default value
         audio                   Enable or disable audio over HDMI (default "on")
         noaudio                 Disable all HDMI audio (default "off")
-        nocomposite             Disable the composite video output (default
-                                "off")
+        composite               Enable the composite output (default "off")
+                                N.B. Disables all other outputs on a Pi 4.
```

Comment:
I'll test, but now that we have the correct VEC 108MHz under all circumstances there's probably no need to disable HDMI on Pi4 with composite.

Reply:
Which would make the only reason to change the default the behaviour of the desktop environment...

Reply:
Doesn't having composite enabled, mean we are outputting a desktop/console picture there, using sdram bandwidth and power. For something 99% of users don't want?

Reply:
Just been chasing this through. Composite-1 reports as "unknown", which DRM takes as "use me as a display of last resort". So normally the console will NOT be using it.

X does seem to enable the output regardless of reported connected state.

Reply:
It didn't stop us in Pi 3 days, when there was less SDRAM bandwidth to go around.